File: twiml/voice/sms/sms-4/sms-4.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        response.Say("Our store is located at 123 Easy St.");
        response.Sms("Store Location: 123 Easy St.",
            statusCallback: new Uri("/smsHandler.php"));

        Console.WriteLine(response.ToString());
    }
}

File: twiml/voice/sms/sms-4/sms-4.5.x.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;


const response = new VoiceResponse();
response.say('Our store is located at 123 Easy St.');
response.sms({
    statusCallback: '/smsHandler.php'
}, 'Store Location: 123 Easy St.');

console.log(response.toString());
